kolla/node_custom_config/horizon/custom_local_settings

```diff
@@ -70,3 +70,36 @@ CHAMELEON_PORTAL_SSO_LOGIN_PATH = '/sso/horizon/'
 CHAMELEON_PORTAL_SSO_LOGOUT_PATH = '/logout/'
 SSO_CALLBACK_HOST = '{{ kolla_external_fqdn }}'
 {% endif %}
+
+# A dictionary of settings which can be used to provide the default values for
+# properties found in the Launch Instance modal.
+LAUNCH_INSTANCE_DEFAULTS = {
+    'config_drive': False,
+    'enable_scheduler_hints': True,
+    'disable_image': False,
+    'disable_instance_snapshot': True,
+    'disable_volume': True,
+    'disable_volume_snapshot': True,
+    'create_volume': False,
+}
+
+# The OPENSTACK_IMAGE_BACKEND settings can be used to customize features
+# in the OpenStack Dashboard related to the Image service, such as the list
+# of supported image formats.
+OPENSTACK_IMAGE_BACKEND = {
+    'image_formats': [
+        ('qcow2', 'QCOW2 - QEMU Emulator'),
+        ('raw', 'Raw'),
+    ],
+}
+
+# The Xen Hypervisor has the ability to set the mount point for volumes
+# attached to instances (other Hypervisors currently do not). Setting
+# can_set_mount_point to True will add the option to set the mount point
+# from the UI.
+OPENSTACK_HYPERVISOR_FEATURES = {
```

Comment:
How does this apply to baremetal? And, this might affect the KVM deployment, which also uses CHI-in-a-box @hammock

Reply:
I tested OPENSTACK_HYPERVISOR_FEATURES and it doesn't work. I can take it out.

Reply:
I would take it out; it will probably mess up the KVM site.

Reply:
KVM overrides this in its own branch. I wouldn't worry about it.